### PR TITLE
Blog App Homework

### PR DESCRIPTION
**Pull request by kaiyusung**
The homework for the blog app.

### CODE REVIEW

File: EdwardSung-kaiyusung/blog_app/blog/app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Comment < ActiveRecord::Base
+  belongs_to :article
```
Comment (tgaff) [inline EdwardSung-kaiyusung/blog_app/blog/app/models/comment.rb]: good, does this make sense now after yesterday's class?

File: EdwardSung-kaiyusung/blog_app/blog/config/routes.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
+Rails.application.routes.draw do
+  get 'welcome/index'
```
Comment (tgaff) [inline EdwardSung-kaiyusung/blog_app/blog/config/routes.rb]: Might want to set this to something else.  It'd be nice to go to the articles index instead

File: EdwardSung-kaiyusung/blog_app/blog/app/assets/stylesheets/custom.scss [hunk 3]
```diff
@@ -0,0 +1,2 @@
+@import "bootstrap-sprockets";
```
Comment (tgaff) [inline EdwardSung-kaiyusung/blog_app/blog/app/assets/stylesheets/custom.scss]: bootstrap looks good on your site

File: EdwardSung-kaiyusung/blog_app/blog/app/controllers/articles_controller.rb [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+
+  #show functions.
+  def show
+      @article = get_article
+  end
+
+  #edit function.
+  def edit
+    @article = get_article
```
Comment (tgaff) [inline EdwardSung-kaiyusung/blog_app/blog/app/controllers/articles_controller.rb]: if you want to stop typing this look up `before_action` which can be used in controllers to run a method before performing the action

**Comment by tgaff** [comment]
It looks like /app/views/comments/_form.html.erb is empty so it renders nothing in your view.  That means users can't make comments! 
You might want something like this:
```
<%= simple_form_for([@article, @article.comments.build]) do |f| %>
  <%= f.input :commenter, label: false, placeholder: 'Your Name (Required)', error: 'Your name is required.' %>
  <%= f.input :body, label: false, placeholder: 'Comment' %>
  <%= f.button :submit, "Post" %>
<% end %>
```

File: EdwardSung-kaiyusung/blog_app/blog/Gemfile [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+# gem 'capistrano-rails', group: :development
+
+#gem files for debugging.
+group :development, :test do
+  gem 'byebug'
+  gem 'pry-rails'
+  gem 'pry-byebug'
+  gem 'pry'
+  gem 'binding_of_caller'
+  gem 'better_errors'
```
Comment (tgaff) [inline EdwardSung-kaiyusung/blog_app/blog/Gemfile]: good, these all look right

File: EdwardSung-kaiyusung/blog_app/blog/app/controllers/comments_controller.rb [hunk 6]
```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  http_basic_authenticate_with name: "name", password: "secret", only: :destroy
+  def create
+    @article = Article.find(params[:article_id])
+    @comment = @article.comments.create(comment_params)
+    redirect_to article_path(@article)s
```
Comment (tgaff) [inline EdwardSung-kaiyusung/blog_app/blog/app/controllers/comments_controller.rb]: this looks like a typo

**Comment by tgaff** [comment]
All in all looks pretty good.  I'm glad you got simple form and bootstrap going.  I think your models are ok, but I can't directly try the comments.  
Good work!  :camel: :cactus:

**Comment by kaiyusung** [comment]
Understood, thank you!...

Yes, this homework made a lot more sense after Monday's class....

**Pull request merged**